org.lvnauth.LVNAuth.json

```diff
@@ -0,0 +1,135 @@
+{
+    "app-id": "org.lvnauth.LVNAuth",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "startup.sh",
+    "finish-args": [
+	    "--socket=x11",
+	    "--socket=fallback-x11",
```

Comment:
thse two are exclusive, the latter being used in combination of wayland. So it doesn't support wayland remove fallback.